configs/packageConfig.js

```diff
@@ -66,9 +120,27 @@ const getBuildOptions = (target) => ({
       category: 'Utility',
       icon: 'resources/icons'
     },
+    publish: {
+      provider: 's3',
+      bucket: 'app-binaries.spacemesh.io'
+    },
     directories: {
       buildResources: 'resources',
       output: 'release'
+    },
+    afterAllArtifactBuild: async (buildResult) => {
+      try {
+        const artifactsToPublishFile = path.join(__dirname, '..', 'release', 'publishFilesList.txt');
+        if (!fs.existsSync(artifactsToPublishFile)) {
```

Comment:
move this outside the callback and put right after declaring "fileHashList" var

Reply:
ok